kpops/component_handlers/streams_bootstrap/app_handler.py

```diff
@@ -21,35 +23,54 @@
 
 
 class AppHandler:
-    def __init__(self, helm_wrapper: HelmWrapper):
-        self._helm_wrapper = helm_wrapper
-        helm_wrapper.helm_repo_add()
+    def __init__(
+        self,
+        helm_config: HelmConfig,
+        helm_repo_config: HelmRepoConfig,
+        helm_diff: HelmDiff,
+    ):
+        self._helm_wrapper = Helm(helm_config)
+        self.repository_name = helm_repo_config.repository_name
+        self.chart_version = helm_repo_config.version
+        self._helm_wrapper.helm_repo_add(
+            helm_repo_config.repository_name,
+            helm_repo_config.url,
+            RepoAuthFlags(
+                username=helm_repo_config.username, password=helm_repo_config.password
+            ),
+        )
+        self.helm_diff = helm_diff
 
     def install_app(
         self,
         release_name: str,
+        application_type: ApplicationType,
```

Comment:
Why do we even use an enum here? In one project we actually introduce a new type and typing is broken because we cannot change the enum.

Reply:
The enum is then used to specify the chart. Because the `install_app` method is called from the producer as well as the streams app. So They need to specify the application that needs to be installed.
```python
        stdout = self._helm_wrapper.upgrade_install(
            release_name=release_name,
            chart=f"{self.repository_name}/{application_type.value}", <------- It is used here!
            dry_run=dry_run,
            namespace=namespace,
            values=values,
            flags=HelmUpgradeInstallFlags(version=self.chart_version),
        )
```
Another approach would be to pass this information true the `HelmRepoConfig` class. WDYT?

Reply:
I think a str would be more appropriate. And we can use the enum values as str constants instead